polynote-frontend/style/styles.less

```diff
@@ -1185,7 +1185,8 @@ body {
       border-bottom: 1px solid @ui-border;
       left: @cell-margin-width;
       width: calc(100% - @cell-margin-width - 1px);
-      overflow: auto;
+      overflow: scroll;
+      max-height: 300px;
```

Comment:
Not sure about this. What if people output charts and stuff? I think there probably ought to be a default `max-height`, but it should probably be pretty big. Like maybe 400pt (~ 5.5in)

Reply:
Good point, removing from this PR.